examples/views/button_roles.py

```diff
@@ -87,7 +87,7 @@ async def post(self, ctx: commands.Context):
 
     @commands.Cog.listener()
     async def on_ready(self):
-        """This function is called every time the bot restarts.
+        """This function is called every time the bot connects to Discord.
```

Comment:
```suggestion
        """This function is called whenever the bot starts up.
```
"connects" isn't correct here, that's the connect event.

Reply:
What about something like "whenever the internal cache and the connection of the bot is ready."?

Reply:
That sounds more likey `is_ready()`,  so I would change it to "whenever the internal cache and the connection of the bot **becomes** ready."

Reply:
Sounds good to me.

Reply:
Maybe "This function is called every time the bot has a proper connection with Discord." or "... the bot has connected to Discord."